src/aten/sycl/CumsumKernel.cpp

```diff
@@ -0,0 +1,38 @@
+// #define TORCH_ASSERT_NO_OPERATORS
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+
+#include <aten/sycl/ScanKernels.h>
+#include <aten/sycl/ScanUtils.h>
+
+namespace at::native {
+
+template <typename In, typename Out>
+struct ScalarConvert {
+  static Out to(const In v) {
+    return (Out)v;
+  }
+};
+
+namespace xpu {
+
+void launch_cumsum_xpu_kernel(
```

Comment:
Let's do right thing, aligning coding style, even not all stock CUDA kernel follow the naming. The naming will be `xxx_kernel`.